salt/minion.py

```diff
@@ -1449,6 +1449,15 @@ def get_executor(name):
             ret,
             timeout=minion_instance._return_retry_timer()
         )
+
+        # Add default returners from minion config
+        # Should have been coverted to comma-delimited string already
+        if isinstance(opts.get('return'), six.string_types):
```

Comment:
@cachedout and @thatch45 , should I pull out this type check? I left it in because if they have a bad type, such as a dict, in the `return` config option I wanted to avoid polluting `data['ret']`